src/i18n/en-AU.properties

```diff
@@ -41,11 +41,11 @@ has_x_refs=This preview has references you cannot view. Open the file in its nat
 
 # Error messages
 # Default preview error message
-error_generic=We're sorry, the preview didn't load.
+error_generic=We’re sorry, the preview didn't load.
```

Comment:
What was wrong with the apostrophes?

Reply:
Normally for user facing strings one uses the accented versions. It also helps with not having the need to escape.